vspace/src/main/java/edu/asu/diging/vspace/web/staff/SlideController.java

```diff
@@ -36,20 +46,31 @@
 
     @Autowired
     private IContentBlockManager contentBlockManager;
+    
+    @Autowired
+    private SpaceRepository spaceRepo;
 
     @RequestMapping("/staff/module/{moduleId}/slide/{id}")
-    public String listSlides(@PathVariable("id") String id, @PathVariable("moduleId") String moduleId, Model model) {
+    public String listSlides(@PathVariable("id") String id, @PathVariable("moduleId") String moduleId, Model model) throws JsonProcessingException {
 
         ISlide slide = slideManager.getSlide(id);
+        List<ISpace> spaces = new ArrayList<ISpace>();
+        spaceRepo.findAll().forEach(s -> {
+            spaces.add(s);
+        });
```

Comment:
why are the spaces added to a new list? doesn't `findAll` return a list?

Reply:
Done